roles/ceph-client/tasks/create_users_keys.yml

```diff
@@ -78,8 +78,13 @@
   command: >
     {{ docker_exec_client_cmd | default('') }} ceph --cluster {{ cluster }}
     osd pool create {{ item.0.name }}
-    {{ item.0.get('pg_num', hostvars[groups[mon_group_name][0]]['osd_pool_default_pg_num']) }}
-    {{ item.0.pgp_num | default(item.0.pg_num) }}
+    {% if groups[mon_group_name] | length > 0 %}
+    {% set _def_pg_num = hostvars[groups[mon_group_name][0]]['osd_pool_default_pg_num'] %}
+    {% else %}
+    {% set _def_pg_num = '' %}
```

Comment:
default to empty?

Reply:
What's a good value in the case where there is no mon and user didn't define pg_num in his pool definition?